### PR TITLE
Add JOOX 1.1.1

### PR DESCRIPTION
**Pull request by PH9**
After making all changes to the cask:

- [X] `brew cask audit --download joox` is error-free.
- [X] `brew cask style --fix joox` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install joox` worked successfully.
- [X] `brew cask uninstall joox` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/joox.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'joox' do
+  version '1.1.1'
+  sha256 '17e7589c0b1f6c19ccc53c850d5546e10bae206e06fe3deebb61853afdf5ce32'
+
+  # download.music.wechat.com/ was verified as official when first introduced to the cask
+  url 'https://download.music.wechat.com/JOOX-1.1.1.dmg?q-sign-algorithm=sha1&q-ak=AKIDxmPp3JKL9kCTYQ5xeGfsDpLitukNh7b6&q-sign-time=1573797970%3B1660197970&q-key-time=1573797970%3B1660197970&q-header-list=&q-url-param-list=&q-signature=dff7699e85f53fbedcc964d1fb7fef3cabd62163'
```
Comment (vladimyr) [inline Casks/joox.rb:6]: @PH9 This URL isn't really permanent?

Comment (PH9) [reply Casks/joox.rb:6]: Yes it is

I have tried on Safari, Google Chrome also in incognito mode and change my IP address it is always the same.

Comment (PH9) [reply Casks/joox.rb:6]: It can be https://www.joox.com/d/desktop/darwin but it will not lock to this version only.

File: Casks/joox.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'joox' do
+  version '1.1.1'
+  sha256 '17e7589c0b1f6c19ccc53c850d5546e10bae206e06fe3deebb61853afdf5ce32'
+
+  # download.music.wechat.com/ was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/joox.rb:5]: Unfortunately, this is what I get when I try to access the page.

image

Which means I’m not able to verify this URL. No HBC maintainer is from one of those regions, as far as I’m aware, so we’re not able to merge this, as a [walled build](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

**Pull request closed without merging**